appstats.js

```diff
@@ -25,7 +25,56 @@ const MAX_STATS_PAGES = 10;
 const STAT_TYPES_LOG = 1;
 
 const DEFAULT_STATS_QUEUE_NAME = 'alertlogic-stats';
+class AzureAppStats {
+    constructor(functionNames = [], _tableService) {
+        this._functionNames = functionNames;
+    }
+    
+    _getFunctionStats(functionName, timestamp, callback) {
```

Comment:
suggest we rename it to `getFunctionStats()`.
Underscore `_` in the function name is suppose to mean a "private" method of a class.